Resources/Prototypes/_Pirate/Entities/Objects/Devices/encryption_keys.yml

```
@@ -0,0 +1,38 @@
+  id: EncryptionKeyInterdynePrivat
+  name: Ключ приватного каналу Інтердайну
+  description: "Здається цей ключ належить корпорації Інтердайн, здається його краще не чіпати"
+  components:
+  - type: EncryptionKey
+    channels:
+    - Interdyne
+    - InterdynePrivat
+    defaultChannel: InterdynePrivat
+  - type: Sprite
+    layers:
+      - sprite: Objects/Devices/encryption_keys.rsi
```

YAMLlint (1.37.1)

[warning] wrong indentation: expected 4 but found 6

(indentation)